packages/infra/lib/api-stack.ts

```diff
@@ -91,6 +92,14 @@ export class APIStack extends Stack {
       subjectAlternativeNames: [`*.${props.config.domain}`],
     });
 
+    const iheCertificate = new cert.PrivateCertificate(this, "IHECert", {
+      domainName: `${props.config.iheSubdomain}.${props.config.domain}`,
+      certificateAuthority: acmpca.CertificateAuthority.fromCertificateAuthorityArn(
+        this,
+        "CA",
+        "arn:aws:acm:us-east-2:463519787594:certificate/10a27170-35eb-4d0e-b232-6e33ca2a7b58"
```

Comment:
see AWS docs for how to create Private Certs: https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_certificatemanager.PrivateCertificate.html

Reply:
Where is this ARN coming from? I couldn't find it on `us-east-2`

Reply:
its the cert for ihe.staging.metriport.com in us-east-2